### DISCUSSION TITLE
[Enhancment/Feature Request] Introduce Z Fade height

### DISCUSSION THREAD

**Discussion by thisisnotfez** [jschuh/klipper-macros#131] [Q&A] [answered]
Absolutely blown away by how much time is being saved by the adaptive mesh sizes for probing. However, I don't see a mesh fade being defined.

If it's there, can you point me to where it is? If not, can it be added?

**Comment by jschuh** [comment, marked as answer]
That's a standard Klipper feature, which you'd enable by adding the appropriate options to your `bed_mesh` config. The [documentation is here](https://www.klipper3d.org/Bed_Mesh.html#mesh-fade).

**Comment by gretel** [comment]
in the meantime: https://github.com/Klipper3d/klipper/commit/5e3daa6f21d6485e4e757d0df00e01a13c968541
